developer_setup.md

```diff
@@ -10,6 +10,7 @@
 
   ```bash
   sudo dnf -y group install "C Development Tools and Libraries"  # For unf Gem and noi4r Gem
+  sudo dnf -y install redhat-rpm-config                             # required for version after Fedora 22
```

Comment:
Why is this required?  I'm on Fedora 27 and don't have this rpm installed.

Reply:
In my case, Fedora 27 Cloud image didn't had this, and was giving error 
`gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory`
So installing this package would fix that.

Reply:
What was being compiled when this failed?  I also don't have this file, but don't remember anything failing.